src/ui/pages/Notifications/NotificationItem.tsx

```diff
@@ -46,6 +47,10 @@ const NotificationItem = ({
         return t("notifications.tab.labels.exnipexapply", {
           connection: connectionsCache?.[item.connectionId]?.label,
         });
+      case ExchangeRoute.IpexGrant:
```

Comment:
This is now mixing `NotificationRoute` and `ExchangeRoute` - I think we need a better way to differentiate between notifications directly from KERIA and ones that we are creating ourselves